exercises/practice/difference-of-squares/difference_of_squares_test.go

```diff
@@ -1,35 +1,44 @@
 package diffsquares
 
-import "testing"
+import (
+	"strconv"
+	"testing"
+)
 
-var tests = []struct{ n, sqOfSum, sumOfSq int }{
-	{5, 225, 55},
-	{10, 3025, 385},
-	{100, 25502500, 338350},
+var tests = []struct{ input, squareOfSum, sumOfSquares int }{
+	{input: 5, squareOfSum: 225, sumOfSquares: 55},
+	{input: 10, squareOfSum: 3025, sumOfSquares: 385},
+	{input: 100, squareOfSum: 25502500, sumOfSquares: 338350},
 }
 
 func TestSquareOfSum(t *testing.T) {
 	for _, test := range tests {
-		if s := SquareOfSum(test.n); s != test.sqOfSum {
-			t.Fatalf("SquareOfSum(%d) = %d, want %d", test.n, s, test.sqOfSum)
-		}
+		t.Run(strconv.Itoa(test.input), func(t *testing.T) {
```

Comment:
Same suggestion as in #2428, I think we could use a test description here and use it as the subtest name.

Reply:
Added descriptions, what do you think about them?